### ISSUE TITLE
Feature Request - Auto Backup to Google Drive and/or Dropbox

### ISSUE THREAD

**Issue by Jackson8890** [federicoiosue/Omni-Notes#616]

**Comment by Pratiwir** [comment]
Yes, but please generalise the backup to any private server. Including .onion address. The backup only neede to be able to not exclude this domain as it resolves to a 10... ip address locally.

**Comment by devtool15424** [comment]
I think it's a good idea in encrypted file to privacy
